moz_security/io_modules/taskcluster.lua

```diff
@@ -0,0 +1,68 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Heka Message Extensions for Taskcluster
+
+This module is intended to be used with another IO module such as a decoder to
+add a username field if the client is a human user.
+
+## Functions
+
+### add_username
+
+Given a Heka message with a Taskcluster clientid, attempt to parse out the username
+associated with the clientid. This is currently specific to Mozilla LDAP usernames,
+as that is the primary way in which human users authenticate with Taskcluster. If no
+username is found, do nothing.
+
+*Arguments*
+- msg (table) - original message
+- field_name (string) - field name in the message to lookup
+
+*Return*
+- none - the message is modified in place if a username is found
+
+## Configuration examples
+taskcluster = {
```

Comment:
i think this needs to be updated for the module cfg / module name change

Reply:
do you mean the change to having `local module_name = ...`? If so, no this is correct. Originally it was `taskcluster_decoder` with `local module_name = "taskcluster_decoder"`

Reply:
right, sorry yeah -- disregard this one